uaclient/upgrade_lts_contract.py

```diff
@@ -42,7 +42,6 @@
     "bionic": "xenial",
     "focal": "bionic",
     "jammy": "focal",
-    "mantic": "lunar",
```

Comment:
should we add oracular while we're here?

Reply:
added